app/src/main/java/de/westnordost/streetcomplete/screens/settings/quest_selection/QuestSelectionList.kt

```diff
@@ -88,7 +88,22 @@ fun QuestSelectionList(
                 bottom = contentPadding.calculateBottomPadding()
             ),
         ) {
-            itemsIndexed(reorderableItems, key = { _, it -> it.questType.name }) { index, item ->
+            itemsIndexed(
+                reorderableItems,
+                key = { _, it -> it.questType.name },
+                contentType = { _, _ -> "QuestSelectionItem" },
+            ) { index, item ->
+
+                val rememberedOnToggleSelection = remember(item.questType) {
+                    { isSelected: Boolean ->
+                        if (isSelected && item.questType.defaultDisabledMessage != 0) {
+                            showEnableQuestDialog = item.questType
+                        } else {
+                            onSelectQuest(item.questType, isSelected)
+                        }
+                    }
+                }
```

Comment:
Since the introduction of [strong skipping](https://developer.android.com/develop/ui/compose/performance/stability/strongskipping#lambda-memoization), it is not necessary anymore to wrap lambdas in a `remember` in order to avoid the lambda being re-created on every recomposition (and thus causing the composable to which the lambda is passed to also recompose).

It is done automatically.